src/oauthClients.json

```diff
@@ -0,0 +1,5 @@
+[{
```

Comment:
particle-CLI is only one client for the server ATM, but it will be more, right? do we will hard code them or implement full CRUD for them?